src/config.rs

```diff
@@ -1742,6 +1743,23 @@ mod tests {
         // Sanity check that key bindings are being parsed
         assert!(!config.key_bindings.is_empty());
     }
+
+    #[test]
+    fn update_dynamic_title() {
+        let config: Config = ::serde_yaml::from_str(ALACRITTY_YML)
+            .expect("deserialize config");
+        assert!(config.dynamic_title);
```

Comment:
I don't think this makes sense. We shouldn't test the default configuration because it's not 'wrong' if we ever switch this to `false`.

Reply:
I'd update the test too personally, but I can remove this check.

Reply:
Yeah I'd just go for writing two tests, one for overriding, one for not doing so. And then scrap this one.